posthog/queries/trends/trends_actors.py

```diff
@@ -77,7 +77,9 @@ def actor_query(self, limit_actors: Optional[bool] = True) -> Tuple[str, Dict]:
             else:
                 breakdown_props = [
                     Property(
-                        key=self._filter.breakdown,
+                        key=f"({self._filter.breakdown}) = '{self._filter.breakdown_value}'"
+                        if self._filter.breakdown_type == "hogql"
+                        else self._filter.breakdown,
```

Comment:
This will work most of the time, but edge cases will either introduce a bug or worse-case a hole.

First, just quote symbols will be problematic. However instead of escaping them with something from `posthog/hogql/escape_sql.py`, I suggest `hogql_context.add_value()`, which will return something like `%(hogql_val_1)`, and get replaced later:


```suggestion
                        key=f"({self._filter.breakdown}) = {self._filter.hogql_context.add_value(self._filter.breakdown_value)}"
                        if self._filter.breakdown_type == "hogql"
                        else self._filter.breakdown,
```

(didn't run the above).

Sometimes the "%" character is also problematic (breaks the python `string.format()` used later on), but in this case I think it was fine, as if a % would have been used within a string in the hogql expression, it would have been val-escaped anyway.

Reply:
Great feedback. Thanks for the explanation. I'll change this accordingly.